src/pages/en/tutorial/5-astro-api/1.md

```diff
@@ -181,19 +183,49 @@ const myPosts = await Astro.glob('../pages/posts/*.md');
 ---
 ```
 
-Write the syntax that represents:
+Choose the syntax that represents:
 
 1. The title of the third blog post returned.  
 
-    || `myPosts[2].frontmatter.title` ||
+ <MultipleChoice>
+    <Option>
+      `myPosts.map((post) => <LastUpdated date={post.frontmatter.pubDate} />)`
+    </Option>
+    <Option isCorrect>
+      `myPosts[2].frontmatter.title`
+    </Option>
+    <Option>
+       `<a href={myPosts[0].url}>First Post!!</a>`
+    </Option>
+</MultipleChoice>
 
 2. The text "First Post!!" linking to the URL of the first blog post returned. 
 
-    || `<a href={myPosts[0].url}>First Post!!</a>` ||
+ <MultipleChoice>
+    <Option>
+      `myPosts.map((post) => <LastUpdated date={post.frontmatter.pubDate} />)`
+    </Option>
+    <Option>
+      `myPosts[2].frontmatter.title`
+    </Option>
+    <Option isCorrect>
+       `<a href={myPosts[0].url}>First Post!!</a>`
+    </Option>
+</MultipleChoice>
 
 3. Returning a `<LastUpdated />` component for each item in `myPosts`, passing the blog post's `pubDate` as its only prop.
```

Comment:
```suggestion
3. Passing the blog post's `pubDate` as its only prop.
```

Reply:
We want to be clear that we're referring to a list - I think we should chop off the last part rather than the first:

> A component for each item in `myPosts`